src/Components/Users/UserProfile.tsx

```diff
@@ -165,14 +166,12 @@ export default function UserProfile() {
             setIsLoading(true);
             const data = {
                 username: username,
-                first_name: states.form.first_name,
-                last_name: states.form.last_name,
+                first_name: states.form.firstName,
+                last_name: states.form.lastName,
```

Comment:
Codacy found an issue: [Identifier 'last_name' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5838477)